[Qt]: fixes m_assumed_blockchain_size variable value

marcoagner opened the pull request:
This is used by Qt but I'm not sure if this is the right tag here.
Please, edit the title if there's something better.

`m_assumed_blockchain_size` (src/chainparams.cpp:CChainParams) was
`BLOCK_CHAIN_SIZE` (src/qt/intro.cpp) and while the transition was being
made by PR 13216 (merged commit: 9d0e528), 3fc2063 changed its value
from 200 to 220, which 9d0e528 ended up reverting.

So, as per MarcoFalke's suggestion (https://github.com/bitcoin/bitcoin/pull/13216#discussion_r247560123), I'm bumping it to 240 before 0.18 is
branched to avoid any confusion.

Anything else (e.g. constexpr) that should/could be done here? Thanks.

MarcoFalke commented:
Tagging 0.18, this should be done prior to a major release: https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#release-process

Bump testnet as well?

laanwj commented:
Yes, probably best to do this just before branching off 0.18, to have the most up-to-date number.

marcoagner commented:
I will then keep this open (if there's no problem, ofc) and rebase it just before branching with the fix/bump of `m_assumed_blockchain_size` for both mainnet and testnet.

laanwj commented:
Sounds good to me!

laanwj commented:
FYI planned split-off date for the 0.18 branch is `2019-03-01`, if you have time please update it on that date or a few days before that

MarcoFalke commented:
I think this can be done right now, we have to add an overhead anyway. Being off by one GB either way won't hurt.

marcoagner commented:
Thank you for the heads up.
I'll be able to update this in a few days when I come back home and access my full node drives to check the exact values plus overhead by myself. (unless somebody suggests one right now, ofc - which happens to be most of the work here haha)

Sjors commented:
Concept ACK. Maybe also add any useful commands to the release process doc.

laanwj commented:
> Being off by one GB either way won't hurt.

Also good point. You'll want to round up anyhow.

MarcoFalke commented:
ACK bd2893be508950d7b47d5bc407c79ed13bfd814e (240GB seems fine, could also bump testnet?)

promag commented:
Concept ACK.

> Maybe also add any useful commands to the release process doc

@Sjors what do you mean?

Sjors commented:
@promag I mean the release process doc could use instructions on how you actually calculate the size correctly.

laanwj commented:
I simply take the size of a synced `~/.bitcoin` and do nothing special to compute it. After all it's used as a guideline for how much space the user needs on their drive, not strictly just the blockchain. I agree documenting that would make sense.

jonasschnelli commented:
Can we also bump Testnet within the same PR?

marcohextor committed:
fixes m_assumed_blockchain_size variables values:

This commit was a fix to `m_assumed_blockchain_size` reverted from
3fc2063's 220 to 9d0e528's 200 since work on 9d0e528 was being done in
parallel and ended up reverting `m_assumed_blockchain_size`.

This commits is now a intended to be a bump of
`m_assumed_blockchain_size` for both mainnet and testnet for new
reasonable values.

marcoagner commented:
8c3fdd3a6d90256bd2d00e86562f4d451bfd9767 now bumps the variable for both mainnet and testnet.
Testnet is now bumped to 30GB; any disagreement to this value?

As for @Sjors's suggestion, it seems proper to document this in another PR (I can update the release doc for this and open the PR, ofc).

fanquake commented:
utACK 8c3fdd3
